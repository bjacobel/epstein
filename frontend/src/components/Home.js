import React, { Component } from 'react';

// import styles from '../stylesheets/main.css';

export default class Home extends Component {
  render() {
    return <></>;
  }
}
