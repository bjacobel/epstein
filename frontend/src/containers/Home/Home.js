import React from 'react';
import { useHistory } from 'react-router';

import SearchBox from '../../components/SearchBox';
import { container, heroCopy } from './style.css';

export default () => {
  const history = useHistory();

  return (
    <div className={container}>
      <p className={heroCopy}>{projectConfig.Description}</p>
      <SearchBox onClick={searchTerm => history.push(`/search/${searchTerm}`)} />
    </div>
  );
};
