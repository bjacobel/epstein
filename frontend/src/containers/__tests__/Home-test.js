import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

describe('home component', () => {
  it('matches snapshot', () => {
    expect(shallow(<Home />)).toMatchSnapshot();
  });
});
