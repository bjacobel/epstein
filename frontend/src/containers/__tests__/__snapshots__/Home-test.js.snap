// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component matches snapshot 1`] = `<Fragment />`;
